component/src/main/java/org/wso2/extension/siddhi/io/websocket/sink/WebSocketSinkHandshakeListener.java

```diff
@@ -43,18 +43,18 @@ public WebSocketSinkHandshakeListener(StreamDefinition streamDefinition, Semapho
     }
 
     @Override
-    public void onSuccess(WebSocketConnection webSocketConnection) {
-        sessionAtomicReference.set(webSocketConnection.getSession());
+    public void onSuccess(WebSocketConnection webSocketConnection, HttpCarbonResponse response) {
+        webSocketConnectionAtomicReference.set(webSocketConnection);
         semaphore.release();
     }
 
-    @Override public void onError(Throwable throwable) {
+    @Override public void onError(Throwable t, HttpCarbonResponse response) {
```

Comment:
Issue with formatting. `@override` should be in a new line.

Reply:
Fixed in 9771fec